### DISCUSSION TITLE
Repeater prevent duplicate

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#6954] [General]
Hello, how can I prevent a repetar to insert data that is already in the json?

![image](https://github.com/filamentphp/filament/assets/111913304/cc6d25b4-b2f2-4223-a68c-267a3e409b61)

for example prevent to add **Taylor Otwell** again on the **add to members** button.

**Comment by pboivin** [comment]
One option is to use a custom validation rule :
https://filamentphp.com/docs/2.x/forms/validation#custom-rules
